src/routes/+page.svelte

```diff
@@ -1,9 +1,12 @@
-<script>
-	import CreateRoom from "$lib/CreateRoom.svelte";
+<script lang="ts">
+	import CreateRoom from '$lib/CreateRoom.svelte';
+	import JoinRoom from '$lib/JoinRoom.svelte';
 
-  let buttonText = "Create a New Room"
+	let buttonText: string = 'Create a New Room';
+	let roomCodePattern: RegExp = /^[A-Z]{6}$/;
```

Comment:
nit: Been a while since I've javascripted, but can these be const?

Reply:
nah you right, I will const these bad boyes up